src/commands/phpmyadmin.ts

```diff
@@ -134,6 +137,7 @@ export class PhpMyAdminCommand {
 		this.track = trackerFn;
 		this.progressTracker = new ProgressTracker( [
 			{ id: this.steps.ENABLE, name: 'Enabling PHPMyAdmin for this environment' },
+			{ id: this.steps.PROCESSING, name: 'Processing' },
```

Comment:
```suggestion
			{ id: this.steps.PREPARING, name: 'Preparing' },
```

I think this sounds better 😅